api/src/main/java/org/openmrs/module/registrationcore/api/RegistrationCoreService.java

```diff
@@ -131,22 +144,52 @@
 	 */
 	List<String> findSimilarFamilyNames(String searchPhrase);
 
+	/**
+	 * Query to MPI server to find patient with Id "patientId".
+	 *
+	 * @param identifier person identifier of patient which should be imported
+	 * @param identifierTypeUuid person identifier type of patient which will be found
+	 * @return found patient
+	 */
+	MpiPatient findMpiPatient(String identifier, String identifierTypeUuid);
+
 	/**
 	 * Query to MPI server to find patient with Id "patientId"
 	 * and save that patient to local DB.
      *
 	 * @param personId person identifier of patient which should be imported
-	 * @return uuid of imported patient
+	 * @return Imported patient
 	 */
-	String importMpiPatient(String personId);
+	Patient importMpiPatient(String personId);
 
-    /**
-     * @return the engine used for biometric operations, if one is enabled
-     */
+	/**
+	 * Query to MPI server to find patient with Id "patientIdentifier" and specific patient identifier type
```

Comment:
I thought this was about importing instead of querying

Reply:
changed for added clarity